server/pulp/server/db/querysets.py

```diff
@@ -116,6 +119,27 @@ def get_or_404(self, **kwargs):
             raise pulp_exceptions.MissingResource(**kwargs)
 
 
+class WorkerQuerySet(CriteriaQuerySet):
+    """
+    Custom queryset for workers
+    """
+
+    def get_workers_online(self):
```

Comment:
What about naming this `get_online()` or just `online()`?

If changed it would read as either `Worker.objects.get_online()` or `Worker.objects.online()`